catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/UdpStreamProcessor.java

```diff
@@ -316,18 +338,40 @@ public PacketBuffer getPacketBuffer() {
      * of IDR boundaries.
      */
     public void shutdown() {
-        try {
-            LOGGER.trace("Shutting down stream processor.");
-            packetBuffer.cancelTimer();
-            streamShutdownPlugin.onShutdown(context);
-        } catch (StreamShutdownException e) {
-            LOGGER.debug("unable to shutdown", e);
-        }
+        LOGGER.trace("Shutting down stream processor.");
+        packetBuffer.cancelTimer();
+
+        Security.runAsAdmin(() -> {
```

Comment:
❓ Do we need to use `runAsAdmin()` here if we execute the code using the `streamCreationSubject` inside the block?